CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Change Log
 
+## [2.0.0] - 2022-08-01
+
+### Changed
+- Breaking change: you now need a `version.txt` file at the root of your repository. This file only contains a version number which is used by taskgraph for release-related operations
+
+### Added
+- `release_notifications` transforms. It enables a task to send e-mail notifications. It usually used in the context of releases when you want to inform a group of people about the completion of a phase of a release. Notifications can contain data about what project/version reached a given phase.
+- New parameters: `version`, `next_version`, `build_number`. As of now, they are used by `release_notification` to send informative e-mails
```

Comment:
Shouldn't this be left up to consumers to define what they want in their notifications?

Reply:
That's some bad wording of mine. Thanks for spotting that! I changed the sentence to be less ambiguous. Let me know if that works better now.